### ISSUE TITLE
API containerwalk-proxies don't actually exist

### ISSUE THREAD

**Issue by corysimmons** [shellscape/postcss-values-parser#34]
Defined here: https://github.com/shellscape/postcss-values-parser/blob/master/API.md#containerwalk-proxies

Don't exist in codebase: https://github.com/shellscape/postcss-values-parser/search?utf8=%E2%9C%93&q=walkAtWords&type=

**Comment by shellscape** [comment]
I really do appreciate your interest in the project, and taking the time to open an issue. However, please do dive into the code just a bit more before pulling the trigger on an issue. Those walkers are registered with [`Container.registerWalker`](https://github.com/shellscape/postcss-values-parser/blob/5e351360479116f3fe309602cdd15b0a233bc29f/lib/container.js#L192). You can find an example of that [here](https://github.com/shellscape/postcss-values-parser/blob/672689884356fe5c0224223d8e30457f075b83a0/lib/colon.js#L13) in the `Colon` class.

**Comment by corysimmons** [comment]
Sorry, how do you use it though?

![image](https://user-images.githubusercontent.com/224636/29010109-f6be95dc-7af6-11e7-9d26-2bf30d966c72.png)

https://github.com/shellscape/postcss-values-parser/blob/master/API.md#containerwalk-proxies could really benefit from a tiny example of usage for those not familiar with proxy functions (like myself). 😬

**Comment by shellscape** [comment]
Try `walkFunctionNodes`. We have a doc discrepancy here due to a reserved word workaround. I also just added some tests confirming the existence of the walkers [here](https://github.com/shellscape/postcss-values-parser/blob/master/test/api.js#L31). I'll double check the API doc to make sure the function names line up. 

As for why they're even there, it was a pattern established by PostCSS that I followed, and they're there as convenience methods for readability as they're a thin layer around `walkType`.
